protocol/communities/manager.go

```diff
@@ -2281,6 +2284,8 @@ func (m *Manager) handleCommunityDescriptionMessageCommon(community *Community,
 		if changes.HasMemberLeft(pkString) {
 			// If we joined previously the community, that means we have been kicked
 			changes.MemberKicked = community.Joined()
+			// soft kick member on community owner change or on ResendAccountsClock change
+			changes.MemberSoftKicked = changes.ControlNodeChanged != nil || prevResendAccountsClock < community.Description().ResendAccountsClock
 		}
```

Comment:
Member can be either kicked or soft kicked, but never both, right? I see the logic as follows:
```suggestion
		if changes.HasMemberLeft(pkString) && community.Joined() {
			if changes.ControlNodeChanged != nil || prevResendAccountsClock < community.Description().ResendAccountsClock {
				changes.MemberSoftKicked = true
			} else {
				changes.MemberKicked = true
			}
		}
```

Reply:
I think it depends on how you threat these flags.
You can say, that the member is kicked or softKicked (MemberKicked || MemberSoftKicked) or you can say that the member is kicked and it is soft kick (MemberKicked && MemberSoftKicked)

It's not a problem to make MemberKicked || MemberSoftKicked only and both can't exist

Reply:
refactored, but in a little bit another way :)